src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is pointing to `main.scss`, which is incorrect. It should point to the compiled CSS file, typically `main.css`, after the SCSS is processed. Ensure that your build process compiles SCSS to CSS and update the link accordingly.